filesystem/File.php

```diff
@@ -85,7 +75,7 @@ class File extends DataObject {
 	private static $db = array(
 		"Name" => "Varchar(255)",
 		"Title" => "Varchar(255)",
-		"Filename" => "Text",
+		"File" => "DBFile",
 		"Content" => "Text",
```

Comment:
I'll have to have a search, but this twingles my smell sensor - is this the Content of the file somehow stored in the DB? (I'm aware this isn't knew, just worried about breakage).